app/Orchid/Layouts/Batch/AddSiteLayout.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Orchid\Layouts\Batch;
+
+use App\Models\Site\Municity;
+use App\Models\Site\Province;
+use App\Models\Site\Region;
+use Orchid\Screen\Field;
+use Orchid\Screen\Fields\Group;
+use Orchid\Screen\Fields\Input;
+use Orchid\Screen\Fields\Relation;
+use Orchid\Screen\Layouts\Rows;
+
+class AddSiteLayout extends Rows
```

Comment:
Should be called `BatchEditSiteLayout`

Reply:
Renamed 'AddSiteLayout' to 'BatchEditSiteLayout'. Also renamed it on the BatchEditScreen.